app/models/office.rb

```diff
@@ -15,4 +15,10 @@ class Office < ActiveRecord::Base
   def managers
     users.where(office_id: id, role: 'manager')
   end
+
+  def business_entities
```

Comment:
This now gives us currently valid `BusinessEntity` objects, which is fine. Will it be a problem when we want to get all of them, including the old ones (ie. for the financial report)?